Yank/commands/analyze.py

```diff
@@ -114,18 +119,67 @@ def dispatch_extract_trajectory(args):
 
 def dispatch_report(args):
     # Check modules for render
+    store = args['--store']
+    output = args['--output']
+    file_full_path, file_extension = os.path.splitext(output)
+    _, file_base_name = os.path.split(file_full_path)
+    # PDF requires xelatex binary in the OS (provided by LaTeX such as TeXLive and MiKTeX)
+    static_extensions = [".pdf", ".html"]
     try:
         import matplotlib
+        import jupyter
     except ImportError:
-        print("Rendering this notebook requires the following packages:\n"
-              " - matplotlib\n"
-              "These are not required to generate the notebook however")
-    store = args['--store']
-    output = args['--output']
+        error_msg = "Rendering this notebook requires the following packages:\n"
+        " - matplotlib\n"
+        " - jupyter\n"
+        "These are not required to generate the notebook however"
```

Comment:
I think it doesn't raise an error because a line with a string is a valid Python expression, but the strings are not concatenated and `error_msg` will be assigned only the string in the first line.